src/test/java/io/confluent/examples/streams/ApplicationResetIntegrationTest.java

```diff
@@ -77,7 +78,11 @@ public void shouldReprocess() throws Exception {
     // Use a temporary directory for storing state, which will be automatically removed after the test.
     streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath());
 
-    KafkaStreams streams = ApplicationResetExample.run(false, streamsConfiguration);
+    streamsConfiguration.put(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG, 1000);
```

Comment:
Why this change? Seems to be unrelated?

Reply:
The other changes are to improve stability in general. This one is specifically addressing a recent test failure in which there was a race between the broker actually completing the cleanup from the reset tool and streams trying to (re)create the internal topics.

Streams only retries 5 times with a backoff time of 100ms, so it only waits 0.5 seconds for the broker to delete the topic, which works some times but not others. Now, it'll wait up to 5 seconds, which should be plenty of time. Of course, if the broker cleans it up sooner, we'll be able to continue the test, in increments of 1 second.